sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Release History
 
+## 12.0.0-beta.5 (2020-11-13)
+
+### New Features
+
+* Support for replaceable HTTP transport layer
```

Comment:
```suggestion
* Support for replaceable HTTP transport layer.
```

Reply:
no period for other sentences. we need to be consistent

Reply:
I agree. I suggest we be consistent **the other way** and add the period (at least going forward).

Look at:
https://github.com/Azure/azure-sdk-for-cpp/releases/tag/azure-core_1.0.0-beta.3
https://github.com/Azure/azure-sdk-for-cpp/releases/tag/azure-storage-files-shares_1.0.0-beta.4
https://github.com/Azure/azure-sdk-for-cpp/releases/tag/azure-storage-blobs_1.0.0-beta.4
https://github.com/Azure/azure-sdk-for-c/releases/tag/1.1.0-beta.2